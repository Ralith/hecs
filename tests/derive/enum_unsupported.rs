use hecs::Bundle;

#[derive(Bundle)]
enum Foo {}

fn main() {}
