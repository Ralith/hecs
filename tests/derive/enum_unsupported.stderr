error: derive(Bundle) does not support enums or unions
 --> $DIR/enum_unsupported.rs:4:6
  |
4 | enum Foo {}
  |      ^^^
